### PR TITLE
Ai-5: Email and email template builder

### PR DESCRIPTION
**Pull request by meladawy**
Here is the issue comments  :- 
image

Changes :- 

BEFORE 
image

AFTER
image

BEFORE
image

AFTER 
image

BEFORE
image

AFTER
image

NOTE:
Styling input file field is not possible as its browser generated and using any tweaks or tricks to style this field would affect on the website compatibility with others browsers as not all browsers support styling this field using -webkit properties

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-email.scss [hunk 1]
```diff
@@ -57,11 +57,29 @@
...
+  .crm-contactEmail-form-block {
+    .form-layout-compressed {
+      width: 100%;
+
+      .crm-contactEmail-form-block-recipient {
+        td {
+          &.label {
+            padding: 30px;
+            font-weight: bold;
+            color: #4D5663;
```
Comment (brunoqueiros) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-email.scss]: Use `$gray-darker`variable.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_accordions.scss [hunk 2]
```diff
@@ -74,6 +74,7 @@
         padding: 15px;
         font-weight: bold;
         text-align: left;
+        color: #4D5663;
```
Comment (brunoqueiros) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_accordions.scss]: Use `$gray-darker`variable.

**Comment by davialexandre** [comment]
@meladawy you're not recognized as the author for the commits in this PR. Since this is already merged,  please fix it for your next PRs

**Comment by meladawy** [comment]
@davialexandre This is resolved man!

**Pull request merged**